database/migrations/2020_07_13_154531_add_stands_to_fpl_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddStandsToFplTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('flight_plans', function (Blueprint $table) {
+            $table->string('dep_stand')->nullable();
+            $table->string('arr_stand')->nullable();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('fpl', function (Blueprint $table) {
+            //
```

Comment:
Should add lines to drop `dep_stand` and `arr_stand` from the table for the down method